packages/wrangler/src/__tests__/pages.test.ts

```diff
@@ -1111,17 +1112,12 @@ describe("pages", () => {
 						// generated and that the file size is greater than zero
 						expect(generatedWorkerJS).not.toBeNull();
 						expect(generatedWorkerJS.size).toBeGreaterThan(0);
-
-						expect(generatedRoutesJSON).toMatchInlineSnapshot(`
-				"{
-				  \\"version\\": 1,
-				  \\"description\\": \\"Generated by wrangler@2.0.28\\",
-				  \\"include\\": [
-				    \\"/hello\\"
-				  ],
-				  \\"exclude\\": []
-				}"
-			`);
+						expect(JSON.parse(generatedRoutesJSON)).toMatchObject({
```

Comment:
We should pull this `JSON.parse(...)` into a variable called something like `maybeRoutesJSONSpec` and wrap with type guards that way we get type errors if the spec ever changes

Reply:
This is a great idea! Will update